samples/multicluster/gen-eastwest-gateway.sh

```diff
@@ -17,6 +17,7 @@
 set -euo pipefail
 
 SINGLE_CLUSTER=0
+REVISION=""
```

Comment:
should we give "default" as the default value?

If so, revision is not strictly required.

Reply:
I'd rather have the zero value so that we don't have to keep defaulting in sync here.

Reply:
if it is 0 value, does it mean user will have to specify this param when installing eastwest gw using the .sh script?